### ISSUE TITLE
Is this wrapper tree shakeable?

### ISSUE THREAD

**Issue by wmonecke** [reactchartjs/react-chartjs-2#693]

**Comment by Luc9802** [comment]
https://bundlephobia.com/package/react-chartjs-2@3.0.3

Bundlephobia says it is 👍

**Comment by daveteu** [comment]
Since it imports everything from chartjs, it's hardly counted as "tree shaken".

https://github.com/reactchartjs/react-chartjs-2/blob/67cd5622d702230f1aa5e06d994134fdd451bd7b/src/index.tsx#L7

If you use chart.js without this package, then you can tree shake the chartjs bundle - https://www.chartjs.org/docs/latest/getting-started/integration.html#bundlers-webpack-rollup-etc

This is me using only Bar and Doughnut of react-chartjs-2 v2.
<img width="609" alt="image" src="https://user-images.githubusercontent.com/18649392/125693679-08d20b37-4698-4621-8367-5277e538aa67.png">

I like this package though.

**Comment by alimirakim** [comment]
Yes @daveteu That's what I discovered as well, and why I came to search Issues if anyone knows if there are plans for this to be updated? We're only using Bar, and our app is already chonky. Being able to shake the rest would save so much.

**Comment by daveteu** [comment]
😅It's working well for me now, I guess I will make do with it until I have extra time on my hand to fork it myself.

**Comment by wmitsuda** [comment]
I hope they prioritize fixing this issue, the bundle size is really too big if you can't tree shake it.
